### PR TITLE
systemd-foo and README improvements

### PR DESCRIPTION
**Pull request by 9R**
added systemd unit file & sysusers.conf to run scraper as systemd.service as dedicated user.

added prometheus job config example & curl debugging example in README

### CODE REVIEW

File: contrib/systemd/prometheus-mumble-exporter.service [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus blackbox exporter for Mumble metrics
+After=network.target nss-lookup.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/conf.d/prometheus-mumble-exporter
+ExecStart=/usr/bin/mumble-exporter $MUMBLE_EXPORTER_ARGS
+User=mumble-exporter
+Group=mumble-exporter
```
Comment (fpletz) [inline contrib/systemd/prometheus-mumble-exporter.service]: Using DynamicUser is more useful than creating a new user for the exporter IMHO. See #1 and https://www.freedesktop.org/software/systemd/man/systemd.exec.html#DynamicUser=.

File: contrib/systemd/prometheus-mumble-exporter.service [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus blackbox exporter for Mumble metrics
+After=network.target nss-lookup.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/conf.d/prometheus-mumble-exporter
```
Comment (fpletz) [inline contrib/systemd/prometheus-mumble-exporter.service]: Which environment variables does this exporter accept?

File: README.md [hunk 3]
```diff
@@ -6,6 +6,24 @@ exports mumble / murmur server stats by using the mumble ping protocol
...
+```
+scrape_config
+ - job_name: 'mumble'
+    scrape_interval: 120s
+    scheme: http
+    params:
+      host: ['mumble.example.com:64738']
+    static_configs:
+      - targets:
+        - 'localhost:8778'
```
Comment (fpletz) [inline README.md]: For these kinds of exporters, a blackbox-like scrape_config with relabelings IMHO makes more sense:

```
scrape_configs:
  - job_name: 'mumble'
    static_configs:
      - targets:
        - mumble.example.com:64738
        - mumble.acme.com:64738
    relabel_configs:
      - source_labels: [__address__]
        target_label: __param_host
      - source_labels: [__param_host]
        target_label: instance
      - target_label: __address__
        replacement: localhost:8778
```

File: contrib/systemd/prometheus-mumble-exporter.service (hunk 2 above)
Comment (9R) [reply contrib/systemd/prometheus-mumble-exporter.service]: missing file added in new commit [96be290](https://github.com/mguentner/mumble_exporter/pull/2/commits/96be290a97bfee3b64dbfd8bb98367f609a48af6)

File: README.md (hunk 3 above)
Comment (9R) [reply README.md]: used improvement in [bbc92e2](https://github.com/mguentner/mumble_exporter/pull/2/commits/bbc92e2d8c168013eab84aa5b79d8cc5d857ffe5)

File: contrib/systemd/prometheus-mumble-exporter.service (hunk 1 above)
Comment (9R) [reply contrib/systemd/prometheus-mumble-exporter.service]: yes. thx for suggestion [48508d8](https://github.com/mguentner/mumble_exporter/pull/2/commits/48508d876710ee3cbab78f6fea8399cfefc9869b)

**Comment by mguentner** [comment]
LGTM :+1:

**Pull request merged**